### PR TITLE
James 2292 get mailqueue

### PR DESCRIPTION
**Pull request by mbaechler**
only last commit to review

### CODE REVIEW

File: server/protocols/webadmin/webadmin-mailqueue/src/test/java/org/apache/james/webadmin/routes/MailQueueRoutesTest.java [hunk 1]
```diff
@@ -136,4 +140,25 @@ public void listAllMailQueuesShouldReturnListWhenSeveralMailQueues() {
         assertThat(actual).containsOnly(FIRST_QUEUE, SECOND_QUEUE, THIRD_QUEUE, FOURTH_QUEUE);
     }
 
+    @Test
+    public void getMailQueueShouldReturnTheMailQueueDataWhenMailQueueExists() throws Exception {
+        MemoryMailQueue queue = mailQueueFactory.createQueue(FIRST_QUEUE);
+        queue.enQueue(Mails.defaultMail().build());
+
+        RestAssured.when()
```
Comment (chibenwa) [inline server/protocols/webadmin/webadmin-mailqueue/src/test/java/org/apache/james/webadmin/routes/MailQueueRoutesTest.java]: static import?

**Comment by chibenwa** [comment]
test this please

```
[fa106f9a1e102327f76a35436fc6e1cc1c920d12]   CassandraMessageSharingTest » OutOfMemory Java heap space
[fa106f9a1e102327f76a35436fc6e1cc1c920d12]   CassandraMessageSharingTest » IllegalState A scoping block is already in progr...
[fa106f9a1e102327f76a35436fc6e1cc1c920d12]   CassandraMessageSharingTest » IllegalState A scoping block is already in progr...
```

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

```
[d89b5ccc93abe2b172252c3bd9f7b0c3eaf77996] Tests in error: 
[d89b5ccc93abe2b172252c3bd9f7b0c3eaf77996]   CassandraMessageSharingTest » OutOfMemory Java heap space
[d89b5ccc93abe2b172252c3bd9f7b0c3eaf77996]   CassandraMessageSharingTest » IllegalState A scoping block is already in progr...
[d89b5ccc93abe2b172252c3bd9f7b0c3eaf77996]   CassandraMessageSharingTest » IllegalState A scoping block is already in progr...
```

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by chibenwa** [comment]
Merged See https://github.com/linagora/james-project/pull/1258

**Pull request closed without merging**